src/couch_mrview/src/couch_mrview_show.erl

```diff
@@ -224,7 +224,8 @@ handle_view_list(Req, Db, DDoc, LName, VDDoc, VName, Keys) ->
             Acc = #lacc{db = Db, req = Req, qserver = QServer, lname = LName},
             case VName of
                 <<"_all_docs">> ->
-                    couch_mrview:query_all_docs(Db, Args, fun list_cb/2, Acc);
+                    ValidatedArgs = couch_mrview_util:validate_all_docs_args(Db, Args),
```

Comment:
This line is also not covered by unit tests.

Reply:
Thank you. It turns out we didn't have any local _list tests, just chttpd ones. So I added a new test suite for local _list functions.